apps/opik-backend/src/main/java/com/comet/opik/api/AlertEventType.java

```diff
@@ -17,7 +17,9 @@ public enum AlertEventType {
     PROMPT_COMMITTED("prompt:committed"),
     TRACE_GUARDRAILS_TRIGGERED("trace:guardrails_triggered"),
     PROMPT_DELETED("prompt:deleted"),
-    EXPERIMENT_FINISHED("experiment:finished");
+    EXPERIMENT_FINISHED("experiment:finished"),
+    TRACE_COST("trace:cost"),
+    TRACE_LATENCY("trace:latency");
```

Comment:
Minor: end lines with a comma so that future changes won't look like existing values are changed
```suggestion
    TRACE_LATENCY("trace:latency"),
    ;
```